src/cookie.js

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) 2015 Florian Hartmann, https://github.com/florian https://github.com/florian/cookie.js
```

Comment:
Can you use `git mv cookie.js src/cookie.js` instead of just `mv`?
That'd make the diffs easier to review and we wouldn't lose any git history on that file
Assuming you haven't made any changes to that file itself, it should also fix the merge conflict.

Reply:
Thanks! Updated: https://github.com/florian/cookie.js/pull/47/commits/5ff762ac9d5cde2cc9023442c34e22fdd8c4b811

Reply:
Thanks!